hooks/useGovernanceAssets.ts

```diff
@@ -680,7 +679,34 @@ export default function useGovernanceAssets() {
       name: 'Token Register Trustless',
       packageId: PackageEnum.MangoMarketV4,
     },
-
+    [Instructions.MangoV4GroupEdit]: {
+      name: 'Group Edit',
```

Comment:
Rename label to `Edit Group`, in the program we named all ixs with the account coming as first word, just so that the files would be arranged/grouped together